helm/hpcc/values.yaml

```diff
@@ -36,6 +36,18 @@ global:
 ## minikube and docker for desktop), or using NFS mounted storage.
 
 storage:
+  planes:
+  # name: default settings
+  #   name: <required>
+  #   prefix: <path>                        # Root directory for accessing the plane (if pvc defined), or url to access plane.
+  #   numDevices: 1                         # number of devices that are part of the plane
+  #   speed: 1000                           # idea is that if the file is on multiple planes, this can help select between them, would (latency, throughput be better?). Higher=better
+  #   replication: nullptr                  # a list or single item indicating which planes the data should be replicated onto
+  #   includeDeviceInPath: false            # Is the device number appended to the mount for the physical path for a file?
```

Comment:
This feels a bit dodgy - if it's something you have to set one way for some types and another way for other types, then it's not something that the helm chart user should be setting up...

Perhaps a philosophical question.

Reply:
I think this is only ever going to be set in weird bare-metal configurations that we don't currently support (to allow local files to be spread over multiple disks.)  Otherwise it is equal to (numDevices != 1)